docs/src/statmodels.md

```diff
@@ -0,0 +1,33 @@
+# Abstraction for Statistical Models
+
+This package defines an abstract type `StatisticalModel`, and an abstract subtype `RegressionModel`.
+
+Particularly, instances of `StatisticalModel` implement the following methods.
+
+```@docs
+adjr2
+aic
+aicc
+bic
+coef
+coeftable
+confint
+deviance
+dof
```

Comment:
Good catch reading the diff, but `df` has been replaced with `dof`, and the RST files hadn't been updated correclty it seems.